### PR TITLE
Only harvest stations passed as parameter in the command line

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
@chicco785 please could you r?

File: Weather/WeatherObserved/harvest/spain_weather_observed_harvest.py [hunk 1]
```diff
@@ -311,7 +320,20 @@ def setup_logger():
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Station Codes to be harvested')
+    parser.add_argument('stations', metavar='stations', type=str, nargs='*', help='Station Code')
```
Comment (chicco785) [inline Weather/WeatherObserved/harvest/spain_weather_observed_harvest.py]: how can i know which codes are available?
how to separate the codes? comma, space?

Comment (jmcanterafonseca) [reply Weather/WeatherObserved/harvest/spain_weather_observed_harvest.py]: here you can find the concerned weather statiosn

https://jmcanterafonseca.carto.com/viz/e7ccc6c6-9e5b-11e5-a595-0ef7f98ade21/public_map

station codes are separated with spaces, this typical argparse Python confoguration

**Comment by jmcanterafonseca** [comment]
the CLI help has been clarified. Issue open to provide README for the harvesters

**Pull request merged**